lib/client/rpc/dao.ts

```diff
@@ -126,5 +128,36 @@ export class FutarchyRPCDaoClient implements FutarchyDaoClient {
       };
     }
   }
+  async getMinLpToProvide(daoAggregate: DaoAggregate): Promise<{ base: BN, quote: BN } | undefined> {
+    const autocrat = new Program<AutocratV1>(AUTOCRAT_V1_IDL, "5scUH1qY87Awh5D8bQkbP2uv9BaXJXiMgrqRBw1K2vVv", this.rpcProvider)
+    const currentDao = daoAggregate.daos.filter((dao) => dao.protocol.deploymentVersion === "V1").slice(-1)[0]
+    if (!currentDao) return
+
+    const daoAccount = await autocrat.account.dao?.fetch(currentDao.publicKey)
+
+    return { base: daoAccount.minBaseFutarchicLiquidity , quote: daoAccount.minQuoteFutarchicLiquidity }
+  }
+
+  async getTreasuryBalance(daoAccount: DaoAccount): Promise<{ total: number, tokens: TokenWithBalance[] }> {
+    const treasury = daoAccount.treasury
+    const balance = await this.rpcProvider.connection.getBalance(treasury).catch(e => null)
+    const tokenAccounts = await this.rpcProvider.connection.getParsedTokenAccountsByOwner(treasury, {
+      programId: new PublicKey("TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA"),
```

Comment:
Just a note here we can import this as well as include token2022 (given that it COULD exist) but this is just a future note.